snuba/datasets/storages/events_common.py

```diff
@@ -107,6 +107,7 @@
         ("platform", String(Modifiers(nullable=True))),
         ("message", String(Modifiers(nullable=True))),
         ("primary_hash", FixedString(32, Modifiers(nullable=True))),
+        ("hierarchical_hashes", Array(FixedString(32))),
```

Comment:
What's the reasoning behind having this be a different column type from the errors storage?

Reply:
I'm just keeping it in sync with `primary_hash` of each table respectively, no particular reason otherwise

Reply:
@lynnagara Is it better to keep these types in sync?

Reply:
This seems fine, although I noticed `primary_hash` in this events_common table is nullable and this is not. Do you know how a situation with a null hash could arise in the first place, and if that could cause any issues here?

Reply:
primary_hash is only null for transaction events. I can represent this with an empty array.